src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/run_conformance.py

```diff
@@ -0,0 +1,222 @@
+        converter_path = os.path.join(self._omz_path, "tools", "model_tools", 'converter.py')
+        mo_path = os.path.join(self._ov_path, "tools", "mo")
+        logger.info(f"Model conversion from {original_model_path} to {converted_model_path} is started")
+        ov_python_path = os.path.join(self._ov_bin_path, "python_api", f"python{version[0:3]}")
+
+        command = f'python3 -m venv .env3; '\
+            f'source .env3/bin/activate; '\
+            f'export OMZ_ROOT={self._omz_path}; ' \
+            f'export PYTHONPATH={ov_python_path}:{mo_path}; ' \
+            f'export LD_LIBRARY_PATH={self._ov_bin_path}; ' \
+            f'pip3 install -e "{mo_path}/.[caffe,kaldi,mxnet,onnx,pytorch,tensorflow2]"; ' \
+            f'pip3 install "{omz_tools_path}/.[paddle,pytorch,tensorflow]"; ' \
+            f'omz_downloader --all --output_dir={original_model_path}; '\
+            f'omz_converter --all --download_dir={original_model_path} --output_dir={converted_model_path}; '\
+            f'source .env3/bin/deactivate'
+        process = Popen(command, shell=True)
+        out, err = process.communicate()
+        if err is None:
+            for line in str(out).split('\n'):
+                logger.info(line)
+        else:
+            logger.error(err)
+            raise Exception("Process failed on step: 'Model conversion'")
+        logger.info(f"Model conversion is successful. Converted models are saved to {converted_model_path}")
+        return converted_model_path
+
+    def download_and_convert_models(self):
+        logger.info("Starting model downloading and conversion")
+        self._omz_path = self.__download_repo(OMZ_REPO_URL, OMZ_REPO_BRANCH)
+        self._model_path = self._convert_models()
+        logger.info("Model downloading and conversion is finished successful")
+
+    def dump_subgraph(self):
+        subgraph_dumper_path = os.path.join(self._ov_bin_path, SUBGRAPH_DUMPER_BIN_NAME)
+        conformance_ir_path = os.path.join(self._working_dir, "conformance_ir")
+        if not os.path.exists(conformance_ir_path):
+            os.mkdir(conformance_ir_path)
+        logger.info(f"Stating model dumping from {self._model_path}")
+        cmd = f'{subgraph_dumper_path} --input_folders={self._model_path} --output_folder={conformance_ir_path}'
+        process = Popen(cmd, shell=True)
+        out, err = process.communicate()
+        if err is None:
+            for line in str(out).split('\n'):
+                logger.info(line)
+            logger.info(f"Conformance IRs were saved to {conformance_ir_path}")
+        else:
+            logger.error(err)
+            raise Exception("Process failed on step: 'Subgraph dumping'")
+        self._model_path = conformance_ir_path
+
+    def _prepare_filelist(self):
+        xmls = Path(self._model_path).rglob("*.xml")
+        filelist_path = os.path.join(self._model_path, "conformance_ir.lst")
+        with open(filelist_path, 'w') as file:
```

Comment:
Need to check. Is it correct behavior if file couldn't be written, but we don't raise an exception? Should we rewrite file without notifications?